math/igraph/Portfile

```diff
@@ -5,6 +5,7 @@ PortGroup           cmake 1.1
 PortGroup           github 1.0
 
 github.setup        igraph igraph 0.9.0
+revision            1
```

Comment:
It sounds like this change only fixes the a build issue on older versions of MacOS (OS X)? If so, the revision should stay at 0 so people who have built it successfully don't have to needlessly rebuild it.

Reply:
This is fixed now.